### ISSUE TITLE
Add a text field for kill causes

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [JoshuaKGoldberg/intern-assassins-2016#44]
"in the library with a spatula"

**Comment by juliusl** [comment]
So I'm thinking the kill cause should be added after the claim is confirmed? Also, should the victim add the cause or the killer?

**Comment by JoshuaKGoldberg** [comment]
> after the claim is confirmed?

Agreed. If the victim reports their own death first we don't want to block the killer from being able to retroactively give the kill info.

> victim or killer?

Previous games I've played have had the killer say it. They're all happy and motivated from the kill while the loser is dejected and unmotivated from having lost.

**Comment by juliusl** [comment]
> They're all happy and motivated from the kill while the loser is dejected and unmotivated from having lost.

Love it. XD

We should add a button for the rest of the living to add a rose to the cause of death notification. Like throwing a rose on the coffin. Just to add more insult to injury.

**Comment by JoshuaKGoldberg** [comment]
Heh. If you want. That would be a good separate issue to file.

**Comment by JoshuaKGoldberg** [comment]
Removing from Day One Features. Not high pri.
